docs/plugins/dynamic_policy_enforcement_points.mdpp

```diff
@@ -3,7 +3,7 @@
 !!! Important
     As of 4.3.0, Static Policy Enforcement Points are now deprecated and will be removed in a later release. For this reason, existing deployments are encouraged to update their policy to use Dynamic Policy Enforcement Points.
 
-iRODS 4.0+ has introduced the capability for dynamic policy enforcement points (PEP).  For every operation that is called, four policy enforcement points are constructed (pre, post, except, and finally varieties), and if it has been defined in core.re or any other loaded rulebase file they will be executed by the rule engine plugin framework.
+iRODS 4.0+ has introduced the capability for dynamic policy enforcement points (PEP).  For every operation that is called, four policy enforcement points are constructed (pre, post, except, and finally), and if a PEP has been defined in `core.re`, or any other loaded rulebase file, it will be executed by the rule engine plugin framework.
```

Comment:
Since we're in here anyway... dunno if this is helpful/accurate:

loaded rulebase file -> configured rulebase file

Reply:
not wrong... is it ... better?   hmmm....

Reply:
Probably better to leave it, thinking on it more.

Reply:
okie doke.